db_main.go

```diff
@@ -100,6 +102,31 @@ func (db *DumbDB) Get(key []byte, bucket string) (ret_val []byte, err error) {
 	return
 }
 
+func (db *DumbDB) GetMultiple(keys [][]byte, bucket string) (ret_val [][]byte, err error) {
+
+	ret_val = make([][]byte, len(keys))
```

Comment:
This is not required. Your slice is a local variable, not a pointer. Append will automatically add more memory. So this make is of no use. Things should work fine even if you remove it.